stix_shifter_modules/tanium/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1013 @@
+{
+    "id": 
+    [
+        {
+            "key": "x-ibm-finding.finding_type",
+            "object": "alert",
+            "transformer": "ReturnAlertTransformer"
+        },
+        {
+            "key": "x-ibm-finding.alert_id",
+            "object": "alert"
+        }
+    ],
+    "eid": 
+    {
+        "key": "x-tanium-alert.eid",
+        "object": "tanium-alert"
+    },
+    "type":
+    {
+        "key": "x-tanium-alert.type",
+        "object": "tanium-alert"
+    },
+    "guid":
```

Comment:
we can map all these generic fields to x-ibm-finding as a custom object such as `x-ibm-finding.x_guid` instead of `x-tanium-alert`